cmake/options.cmake

```diff
@@ -1,27 +1,13 @@
-option(mpi "use MPI library" off)
-option(openmp "use OpenMP" off)
-option(zlib "build ZLIB" off)
-option(json "build Jansson" off)
+option( SC_ENABLE_MPI "use MPI library" OFF )
+option( SC_ENABLE_OPENMP "use OpenMP" OFF )
 
-option(SC_BUILD_TESTING "build libsc self-tests" on)
-option(TEST_WITH_VALGRIND "run self-tests with valgrind" OFF)
-option(BUILD_SHARED_LIBS "build shared libsc")
-option(CMAKE_TLS_VERIFY "verify TLS certificate" on)
+option( SC_USE_INTERNAL_ZLIB "build ZLIB" OFF )
+option( SC_USE_INTERNAL_JSON "build Jansson" OFF )
 
-# --- default install directory under build/local
-# users can specify like "cmake -B build -DCMAKE_INSTALL_PREFIX=~/mydir"
+option( SC_BUILD_SHARED_LIBS "build shared libsc" OFF )
+option( SC_BUILD_TESTING "build libsc self-tests" ON )
+option( SC_TEST_WITH_VALGRIND "run self-tests with valgrind" OFF )
```

Comment:
If changing this variable name, cmake/valgrind.cmake and test/CMakeLists.txt should be updated as well.

Reply:
Thanks for letting me know. This should be fixed now